Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "nasu"
 version = "0.0.2"
-authors = ["Esteban Borai <estebanborai@gmail.com>", "David Camperos"]
+authors = ["Esteban Borai <estebanborai@gmail.com>", "David Camperos <camperos.davidjosue@gmail.com"]
```

You are missing the closing greater-than sign for tour email here!

```suggestion
authors = ["Esteban Borai <estebanborai@gmail.com>", "David Camperos <camperos.davidjosue@gmail.com>"]
```